railties/lib/rails/application/configuration.rb

```diff
@@ -53,6 +53,7 @@ def initialize(*)
         @x                               = Custom.new
         @enable_dependency_loading       = false
         @read_encrypted_secrets          = false
+        @secrets_cipher                  = "aes-128-gcm"
```

Comment:
Is it better to leave it here and not set it in the bootstrap (to keep defaults all in the same place)?